Sources/tart/Commands/Prune.swift

```diff
@@ -63,21 +63,22 @@ struct Prune: AsyncParsableCommand {
     let prunableStorages: [PrunableStorage] = [VMStorageOCI(), try IPSWCache()]
     let prunables: [Prunable] = try prunableStorages
             .flatMap { try $0.prunables() }
-            .sorted { try $0.accessDate() < $1.accessDate() }
+            .sorted { try $0.accessDate() > $1.accessDate() }
 
-    let cacheUsedBytes = try prunables.map { try $0.sizeBytes() }.reduce(0, +)
-    var cacheReclaimedBytes: Int = 0
+    var cacheBudgetBytes = cacheBudgetBytes
+    var prunablesToDelete: [Prunable] = []
 
-    var it = prunables.makeIterator()
+    for prunable in prunables {
+      let prunableSizeBytes = UInt64(try prunable.sizeBytes())
 
-    while (cacheUsedBytes - cacheReclaimedBytes) > cacheBudgetBytes {
-      guard let prunable = it.next() else {
-        break
+      if prunableSizeBytes <= cacheBudgetBytes {
+        cacheBudgetBytes -= prunableSizeBytes
```

Comment:
Let's add a comment so it's easier to understand. Maybe something like:

```suggestion
        // don't mark for deletion while there is budged
        cacheBudgetBytes -= prunableSizeBytes
```

Reply:
See https://github.com/cirruslabs/tart/pull/272/commits/011369d2148d8d0d0d2f00cdb7a9f0d2d6fc7116.